xlators/mgmt/glusterd/src/glusterd-mgmt.c

```diff
@@ -2201,6 +2201,8 @@ glusterd_mgmt_v3_post_validate(glusterd_op_t op, int32_t op_ret, dict_t *dict,
                  "to %d peers. Returning %d",
                  gd_op_list[op], peer_cnt, ret);
 out:
+    if (rsp_dict)
+        dict_unref(rsp_dict);
```

Comment:
The `rsp_dict` is barely used. Why not just move lines 2151 and 2152 before line 2132 ?

Reply:
done